pkg/cluster/cluster_config.go

```diff
@@ -257,5 +265,40 @@ func getClusterInfo(ctx context.Context, cli client.Client) (ClusterInfo, error)
 	}
 	c.Version = ocpVersion
 
+	// Check for FIPs
+	if fipsEnabled, err := IsFipsEnabled(ctx, cli); err == nil {
+		c.FipsEnabled = fipsEnabled
+	} else {
+		logf.FromContext(ctx).Info("could not determine FIPS status, defaulting to false", "error", err)
```

Comment:
if we fail to get value from configmap, we default bool to false for FipsEnabled, but no error only info ?